src/components/widget.js

```diff
@@ -160,18 +164,24 @@ class Widget extends LitElement {
 
           <!-- Organization -->
           ${
-            this.member.organization &&
-            html`
+            !!this.member.organization
+              ? html`
             <div class="flex flex-row justify-start items-center mt-1">
               ${
                 this.member.organization.logo_url &&
-                html` <img
+                html`<img
                   class="mr-1 w-5 h-5"
                   src="${this.member.organization.logo_url}"
                 />`
               }
+              <!-- If organisation doesn't include a protocol (ie https://),
+              prepend it with // so it's treated as absolute regardless -->
               <a
-                href="${this.member.organization.website}"
+                href="${
+                  this.member.organization.website.match("//")
+                    ? this.member.organization.website
+                    : `//${this.member.organization.website}`
```

Comment:
So `//` will preserve the current protocol for the new link. I used it instead of specifying https because I was worried about... well, what if for some godforsaken reason a user was only using http? But, yeah, realistically this shouldn't really be an issu. Will update & make the matcher more strict as well ^^